packages/ui-transfer-components/lib/components/loading.component.ts

```diff
@@ -16,6 +18,7 @@ export class LoadingComponent extends RxLitElement {
     return html`
       <div class="loading-container">
         ${unsafeSVG(Loading)}
+        ${this.message ? html`<p> ${this.message} </p>` : ''}
```

Comment:
Sorry, still would prefer undefined (and no extra spaces around the message).
```suggestion
        ${this.message ? html`<p>${this.message}</p>` : undefined}
```